Simulator/components/AddressBar.jsx

```diff
@@ -1,4 +1,5 @@
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
```

Comment:
If we have .jsx changes, do we need to do bundle and wdc-simulator.js gets its updates?

Reply:
Changing the `prop-types` import to this style will allow support without breaking the build should we upgrade the version of `react` to > 15 in the future. I've updated `wdc-simulator.js` with `npm run bundle` and added to the PR.